uint8 my_counter 
float32 status
